content/en/docs/contribute/style/style-guide.md

```diff
@@ -38,29 +38,25 @@ The English-language documentation uses U.S. English spelling and grammar.
+  projects.
 
-* Vendor, provider, or project-specific content from projects that are not:
-
-    * Inside the kubernetes or kubernetes-sigs GitHub organizations
-    * Inside the CNCF
-
-* Identical content hosted in multiple locations ("dual-sourced" content)
+Kubernetes documentation does not host identical content sourced from
+ multiple locations ("dual-sourced" content).
 
 For example:
 
-| Allowed |                           Action                          |                                                    Reason                                                   |
-|:-------:|:----------------------------------------------------------|:------------------------------------------------------------------------------------------------------------|
-|    Y    | Linking to Kubernetes in Docker (KinD) docs from k8s docs | The KinD project resides in github.com/kubernetes-sigs.                                                     |
-|    N    | Hosting KinD docs in k8s website                          | Dual-sourced content requires duplicated effort from maintainers and tends to go stale/rotten more quickly. |
-|    Y    | Linking to Prometheus docs from k8s docs                  | Prometheus is a CNCF project.                                                                                |
-|    N    | Linking to rkt docs from k8s docs                         | rkt is an (almost) archived CNCF project.                                                                    |
-|    N    | Tutorial on using a specific product to monitor Kubernetes | Third-party product-specific content not allowed; this type of content belongs in the third-party product's documentation. |
-|    Y    | Linking to vendor-specific product docs when listing third-party products such as Ingress controllers, load-balancing products, monitoring tools, and device plugins | Avoids dual-sourced content
-|    Y    | Linking to an online training course                      |  Avoids dual-sourced content                                                                                  |
-
+| Allowed |                           Action                                              |                                                    Reason                                                   |
+|:-------:|:------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------|
+|    Yes  | Linking to Kubernetes in Docker (KinD) docs from k8s docs                     | The KinD project resides in `kubernetes-sigs` GitHub organization.                                                     |
+|    No   | Hosting KinD docs in k8s website                                              | Dual-sourced content requires duplicated effort from project maintainers and tends to go stale/rotten more quickly. |
```

Comment:
nit: The `Action` column could be the first column.
nit: The table could use an intro paragraph explaining the rules for contributing acceptable content. The table then provides examples of this content.
nit: Hosting [KinD](...) docs `from the` Kubernetes website; link to KinD?
nit: Line 59, Is linking to a CNCF or Linux Foundation online training course permitted?
nit: Kubernetes docs vs Kubernetes Docs?

Reply:
RE inking to a CNCF or Linux Foundation online training course permitted - I would say no, because those links can and do go stale. Courses are removed from current offerings, updated with a name change, and the like.

Reply:
@kbhawkey excellent eye! Thanks for the feedback!